README.md

```diff
@@ -435,8 +435,6 @@ Below are listed all the variables you can customize (you may also want to take
         host: ""
         port: ""
         options: {}
-    # SPATIALITE_LIBRARY_PATH django setting
-    openwisp2_spatialite_path: "mod_spatialite"
```

Comment:
Since `variable.yml` runs everytime and the value set by the file takes precedence over value in `playbook.yml`; this variable will have no effect.
If we want `playbook.yml`'s values to have precedence, we'll need to take `openwisp2_spatialite_path` to a file with lower precedence, should i do that?

Reply:
After, `openwisp2_spatialite_path: false` is added to `default/main.yml` and condition to not change value from `variables.yml` for RedHat system is added. This value from `playbook.yml` will take effect in RedHat systems, but this will be a little inconsistent because the value will still be obsolete for other OS.

Reply:
you're right, so we could put `null` instead and execute `set_fact` only if `openwisp2_spatialite_path == null`

Reply:
That should do the trick, thanks, i am on it! :smile: